common/src/main/java/com/wynntils/models/activities/ActivityModel.java

```diff
@@ -40,7 +40,7 @@
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-public final class ContentModel extends Model {
+public final class ActivityModel extends Model {
```

Comment:
Add a comment saying this is wynn's content book basically?

Reply:
Good point.